assignment6.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+from statistics import median
+import unittest
+import numpy as np
+
+class RearrangingArray():
+
+	def __init__(self, arrangement, verbose=False):
```

Comment:
I was thinking maybe would be useful to explain a little bit the arguments from the 'init'